docs/modules/migrate/pages/lts.adoc

```diff
@@ -0,0 +1,130 @@
+= Long-term Support Releases
+
+:description: Hazelcast simplifies your upgrade experience with the introduction of long-term support (LTS) releases. You can upgrade directly from a supported previous release to the LTS release or directly between LTS releases using a rolling upgrade.
+:page-enterprise: true
+
+{description}
+
+LTS releases have been introduced with 5.5. You can upgrade directly from 5.2.x or later to 5.5 without the need to upgrade to each release between your current version and the LTS release. We recommend that you upgrade to 5.5 from the latest available patch for your current release.
+
+NOTE: Features marked as Beta in the source version are upgraded, but are supported only in a cluster of the source version due to changes that can occur between releases. For further information on rolling upgrade compatibilities, see xref:maintain-cluster:rolling-upgrades.adoc#hazelcast-members-compatibility-guarantees[Compatibility Guarantees].
+
+Between LTS releases, Hazelcast will provide short-term support (STS) releases. You can choose to upgrade directly from one LTS to the next LTS, or you can upgrade from one LTS to each STS release and then from the last STS release to the next LTS release. Before deciding the approach that best suits your needs, consider the following:
+
+[cols="1,1"]
+|===
+|LTS|STS
+
+|Focus on stability
+|Focus on innovation
+
+|Consolidates the functionality introduced in all STS releases since the last LTS release. Can include improvements, such as increased capacity limits
+|Provides the latest features and bug fixes
+
+|Released every two years
+|Released at least twice each year
+
+|Upgrade directly from the previous LTS, or from the STS release that precedes the latest LTS release
```

Comment:
> Upgrade directly from the previous LTS, or from the STS release that precedes the latest LTS release

@frant-hartm should this be "Upgrade directly from the previous LTS" only? I think for 5.5 we are making an exception and make it work for "Upgrade directly from the previous LTS, or from the STS release that precedes the latest LTS release"

Reply:
But surely there is no previous STS or LTS for 5.5 - and we have covered this in the remaining text? 

This table covers the logic/behaviour for LTS/STS releases as they will be - it was difficult to convey the idea without getting bogged down in the initial implementation where there are no other LTS or STS releases avaialbe.

We could be specific for this release and say something like:

> For the initial LTS release with 5.5, you can upgrade directly from 5.2.x or later. 
> 
> In future LTS releases, you can <existing wording>

Of course, this then raises the question of whether we need to add something like (when available) or (available from release ????) in the STS header. 

Then this additional wording can be stripped as we progress through the availability of STS releases and further LTS releases.